prisma/schema.prisma

```diff
@@ -12,3 +12,37 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+enum ContentType {
+  onboardingPrompts
+  blogHome
+  pledge
+  goodPartyTeamMembers
+  articleCategories
+  faqArticles
+  articleTags
+  recentGlossaryItems
+  aiChatPrompts
+  elections
+  redirects
+  glossaryItems
+  promptInputFields
+  glossaryItemsByTitle
+  aiContentCategories
+  privacyPage
+  candidateTestimonials
+  contentPromptsQuestions
+  blogSections
+  glossaryItemsByLetter
+  candidateContentPrompts
+  blogArticles
+  termsOfService
+}
+
+model Content {
+  createdAt BigInt      @map("created_at")
+  updatedAt BigInt      @map("updated_at")
+  id        String      @id
+  type      ContentType
```

Comment:
can we add an index here?

Reply:
On the type?

Does that make sense? The types here aren't unique.

When I mentioned we could do that, I was talking about adding an index on unique lookup properties on the JSONb `data` column (i.e. 'slug' for `blogArticle` content records).  But also, I also was saying we _can_ do that, if we find we need it.

ATM we don't have that many records in Contentful that I think adding an index here would be perceptible, do you think?

Reply:
it is the main way we access this record, and contentful keeps growing. It will reduce the lookup time from O(n) to O(logN), so why not?
the column doesn't have to be unique to be indexed; this is not a primary key, of course.